README.md

```diff
@@ -52,6 +97,31 @@ $ npm run server
 
 and navigate to: http://127.0.0.1:3001/test/func/fixtures/
 
+### Releases
+
+**IMPORTANT - NPM**: To correctly run `preversion` your first step is to make
```

Comment:
What is `preversion`? Also, what is "a very modern `npm` binary". Can't you just specify that this works for `npm@2` or `npm@3` instead?